Documentation/development-flow.md

```diff
@@ -52,6 +52,65 @@ git remote -v
 ```
 Now you should have at least `origin` and `upstream` remotes. You can also add other remotes to collaborate with other contributors.
 
+## Layout
+
+A source code layout is shown below, annotated with comments about the use of each important directory:
+
+rook  
```

Comment:
this looks weird when rendered in a browser, e.g. https://github.com/rook/rook/blob/f9d20e573d3fd7f71087e44cf8dd7725d1acfb74/Documentation/development-flow.md#layout

perhaps enclose it in a triple back-tick code block?

Reply:
Updated